industrial_ci/src/tests/source_tests.sh

```diff
@@ -117,7 +117,12 @@ fi
 
 ici_time_start rosdep_install
 
-rosdep install -q --from-paths $CATKIN_WORKSPACE --ignore-src --rosdistro $ROS_DISTRO -y
+function rosdep_install {
```

Comment:
Actually this is a little hack to make the job fail if rosdep fails.
As an alternative I could have surrounded it with
```
set -o pipefail
set +o pipefail
```

Hwoever, I plan to factor all code into functions anyway.